README.md

```diff
@@ -82,6 +82,10 @@ The output C# code files will go in to the `output` directory.
 
 `.\typewriter.exe -v Info -m D:\v1.0_2018_10_23_source.xml -o output -l CSharp -d D:\repos\microsoft-graph-docs -g Full`
 
+## Use/debug Typewriter in VSCode
+- Prerequisite is that `msgraph-metadata` is checked out at the same root folder as `MSGraph-SDK-Code-Generator`.
+- `launch.json` file contains main generation scenarios for each language/endpoint pair. Running Typewriter using one of those generation scenarios (e.g. `Generate CSharp beta`) will output the corresponding models into a folder, which will be a sibling to `MSGraph-SDK-Code-Generator`. The folder will be named as `generator-output-<language>-<endpoint>` e.g. `generator-output-CSharp-beta`.
```

Comment:
why don't we use the repos names/paths for the output instead of those placeholders?

Reply:
See #514

Reply:
so in that model we'd have the debug process to generate to an output folder and a post script to delete + move things, correct? I think launch.json supports adding such post scripts, would you mind adding it as well?

Reply:
Linked this comment to be addressed as part of #514.